src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+// const div = document.getElementById('div');
+const logo = document.getElementById('logo');
+
+const addMessage = (text, className, className2) => {
+  const div = document.createElement('div');
+
+  document.body.append(div);
+  div.dataset.qa = 'notification';
+  div.classList.add(className);
+  div.classList.add(className2);
+  div.textContent = text;
+};
```

Comment:
Lets call it more descriptively, also you can pass multiple class name in one string:
```suggestion
const printMessage = (text, isError = false) => {
  const div = document.createElement('div');

  div.dataset.qa = 'notification';
  div.classList.add("message");
  isError && div.classList.add("error-message");
  div.textContent = text;
  document.body.append(div);
}
```